addons/web/static/src/views/widgets/widget.js

```diff
@@ -75,7 +75,7 @@ Widget.parseWidgetNode = function (node) {
         options: evaluateExpr(node.getAttribute("options") || "{}"),
         name,
         rawAttrs: attrs,
-        WidgetComponent,
+        fieldDependencies,
```

Comment:
for the sake of consistency with field, I would add the whole widget (from the registry) here, not only fieldDependencies